docs/docs/building.md

```diff
@@ -27,89 +26,63 @@ contract MyContract is Migratable {
 }
 ```
 
-You might have noted that our sample contract has an `initialize` function instead of the standard contstructor. This is a requirement of ZeppelinOS's upgradeabiltiy system, you can learn more about this in our [Advanced topics](advanced.md) section. 
+Notice that our sample contract has an `initialize` function: this works as a constructor for our upgradeable contract. This is a requirement of the [ZeppelinOS's upgradeability system](advanced.md#initializers-vs-constructors).
 
-You can now compile the contract with:
+Now let's compile the contract:
 
-```
+```sh
 npx truffle compile
 ```
 
-## Initial Deployment
+## Initial deployment
 
-Register all the contract implementations of the first version of your
-project running:
+To deploy our app, we need to register the first version of our contract:
 
+```sh
+zos add MyContract
 ```
-zos add <contract_name_1>
-zos add <contract_name_2>
-...
-zos add <contract_name_n>
-```
-
-In our example, run:
 
-    zos add MyContract
+> **Note**: If you are working in a local development network like [Ganache](http://truffleframework.com/ganache/), you will need to [configure](http://truffleframework.com/docs/getting_started/project#alternative-migrating-with-ganache) your `truffle.js` file before running the `push` command.
 
-If you are working in a local development network like [Ganache](http://truffleframework.com/ganache/), you will need to [configure](http://truffleframework.com/docs/getting_started/project#alternative-migrating-with-ganache) your `truffle.js` file before running the `push` command.
+We can now push our application to the network by running:
 
-You can now push your application to the desired network by running:
-
-```
+```sh
 zos push --network <network>
 ```
 
-which will create a `zos.<network>.json` file with all the information specific to the chosen network. The format of this file is discussed in the [Advanced topics](advanced.md) section.
+This  will create a `zos.<network>.json` file with all the information specific to the chosen network. You can read more about this file in the [advanced topics](advanced.md#format-of-zosjson-and-zos-network-json-files) section.
```

Comment:
Revert last part:
You can read more about this file -> The format of this file is discussed